src/Chapter1.hs

```diff
@@ -508,7 +509,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = min (abs x) (abs y)
```

Comment:
This solution is almost correct :ok_hand: However, on negative numbers, it returns the absolute value of those negative numbers, note the value itself.